config.py

```diff
@@ -12,17 +12,17 @@
 
 module_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "modules")
 
-def get_all_modules() -> set[str]:
-    modules = []
+def get_all_modules() -> frozenset[str]:
+    modules = set()
     for file_name in os.listdir(module_dir):
         if file_name.endswith('.py') and file_name != '__init__.py':
-            modules.append(file_name[:-3])
+            modules.add(file_name[:-3])
 
-    return set(modules)
+    return frozenset(modules)
 
 All_Stampy_Modules = get_all_modules()
 
-def getenv(env_var: str, default: Optional[str] = NOT_PROVIDED) -> Optional[str]:
+def getenv(env_var: str, default: Any = NOT_PROVIDED) -> Optional[Any]:
```

Comment:
what else than `Optional[str]` can it be?

Reply:
also, `Any` includes `None`, so `Optional[Any]` is redundant

Reply:
I mean, you could put a `default` value which is not `str` and then this would make sense but I don't think  it's a good idea to make this function return one type if it finds the key in env and another type (except `None`) if it doesn't find it.

If we actually ever need it, I'd rather do something like

```python
x = getenv("k") # str
if x is None:
    x = default # some other type
```
or even

```python
x = getenv("k") or default
```

but I think this is very rarely a good idea

Reply:
I'll try to implement Matt's suggestion